src/components/alerting.tsx

```diff
@@ -0,0 +1,2167 @@
+import {
+  Action,
+  ActionServiceProvider,
+  Alert,
+  AlertSeverity,
+  AlertStates,
+  BlueInfoCircleIcon,
+  consoleFetchJSON,
+  GreenCheckCircleIcon,
+  ListPageFilter,
+  PrometheusAlert,
+  PrometheusEndpoint,
+  PrometheusLabels,
+  RedExclamationCircleIcon,
+  ResourceLink,
+  ResourceStatus,
+  RowFilter,
+  RowProps,
+  Rule,
+  Silence,
+  SilenceStates,
+  TableColumn,
+  Timestamp,
+  useActivePerspective,
+  useListPageFilter,
+  VirtualizedTable,
+  YellowExclamationTriangleIcon,
+} from '@openshift-console/dynamic-plugin-sdk';
+import {
+  Alert as PFAlert,
+  Breadcrumb,
+  BreadcrumbItem,
+  Button,
+  CodeBlock,
+  CodeBlockCode,
+  Dropdown,
+  DropdownItem,
+  DropdownPosition,
+  DropdownToggle,
+  Flex,
+  FlexItem,
+  KebabToggle,
+  Label,
+  Modal,
+  ModalVariant,
+  Popover,
+  Toolbar,
+  ToolbarContent,
+  ToolbarGroup,
+  ToolbarItem,
+} from '@patternfly/react-core';
+import {
+  BanIcon,
+  BellIcon,
+  BellSlashIcon,
+  HourglassHalfIcon,
+  OutlinedBellIcon,
+} from '@patternfly/react-icons';
+import { sortable } from '@patternfly/react-table';
+import classNames from 'classnames';
+import i18next from 'i18next';
+import * as _ from 'lodash-es';
```

Comment:
Should we refactor the usages to avoid importing the whole library?

Reply:
It may be worth doing that yes. However, lodash is used a lot in this file so I think it's better to make it a separate PR.

Reply:
I added a task for this: https://issues.redhat.com/browse/OU-139